extensions/wikia/SpecialDiscussionsLog/SpecialDiscussionsLog.setup.php

```diff
@@ -17,6 +17,9 @@
 $wgAutoloadClasses['SpecialDiscussionsLogController'] = $dir . 'SpecialDiscussionsLogController.class.php';
 $wgAutoloadClasses['UserLogRecord'] = $dir . 'UserLogRecord.class.php';
 
+// hooks
+$wgHooks['ContributionsToolLinks'][] = 'SpecialDiscussionsLogController::onContributionsToolLinks';
```

Comment:
Since this is a hook we're adding, I believe the convention is to add it to a dedicated hooks class. Something like `SpecialDiscussionsLogHooks`. Can we do that here?